backend/main.py

```
@@ -234,45 +230,39 @@ async def create_issue(
-        "id": new_issue.id,
-        "message": "Issue reported successfully",
-        "action_plan": action_plan
-    }
+        new_issue = await asyncio.to_thread(save_to_db)
+
+        return {
+            "id": new_issue.id,
+            "message": "Issue reported successfully",
+            "action_plan": action_plan
+        }
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)